From 0d8e582ac8daa0e6afa7ef8b04de3a3d2a5f4b5e Mon Sep 17 00:00:00 2001
From: vulpes <kazz9000@gmail.com>
Date: Thu, 12 Oct 2023 18:48:39 +0900
Subject: [PATCH] Update .gitignore

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index de516a2..6933f9d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -4,6 +4,7 @@ spring-security.xml
 servlet-context.xml
 root-context.xml
 mybatis-config.xml
+web.xml
 
 /target/
 !.mvn/wrapper/maven-wrapper.jar
-- 
2.39.3.windows.1

